### PR TITLE
create order

### PR DESCRIPTION
**Pull request by datnguyen0126**

### CODE REVIEW

**Comment by datnguyen0126** [comment]
[Create orders](https://trello.com/c/LGiSCDPu/19-create-orders)

File: FoodsAndDrinks/settings.py [hunk 1]
```diff
@@ -175,3 +176,11 @@
 
 CART_SESSION_ID = 'cart'
 
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_PORT = 587
```
Comment (prd-anh-nguyen) [inline FoodsAndDrinks/settings.py:181]: please add to enviroment and have default value

File: orders/models/OrderItemModel.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from foods.models.FoodModel import Food
+from .OrderModel import Order
+
+class OrderItem(models.Model):
+    order = models.ForeignKey(Order, related_name='items', on_delete=models.CASCADE)
+    food = models.ForeignKey(Food, on_delete=models.CASCADE)
+    order_price = models.DecimalField(max_digits=10, decimal_places=2)
```
Comment (prd-anh-nguyen) [inline orders/models/OrderItemModel.py:8]: `order_price` -> `price`

File: orders/models/OrderModel.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    address = models.CharField(max_length=250)
+    city = models.CharField(max_length=100)
+    contact_number = models.IntegerField()
+    created_at = models.DateTimeField(auto_now_add=True)
+    delivery_at = models.DateTimeField(null=True)
+    status = models.CharField(max_length=100, default=1)
+
+    class Meta:
+        ordering = ('-created_at',)
```
Comment (prd-anh-nguyen) [inline orders/models/OrderModel.py:16]: ('-created_at')

File: orders/views.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+from .models.OrderModel import Order
+from .tasks import order_created
+from django.http import JsonResponse
+from requests.exceptions import HTTPError
+
+
+User = get_user_model()
+
+class OrderViewSet(viewsets.GenericViewSet):
+    permission_class = [IsAuthenticated, ]
```
Comment (prd-anh-nguyen) [inline orders/views.py:20]: `[IsAuthenticated]`

File: orders/views.py [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    serializer_class = serializers.OrderSerializer
+    
+    def create(self, request):
+        cart = Cart(request)
+        order_serializer = serializers.OrderSerializer(data=request.data, context={'request': request})
+        order_serializer.is_valid(raise_exception=True)
+        order = order_serializer.save()
+        for item in cart(str(request.user.id)):
+            OrderItem.objects.create(order=order, food=item['food'], order_price=item['price'], quantity=item['quantity'])
```
Comment (prd-anh-nguyen) [inline orders/views.py:29]: what happend when a order item create false?

**Pull request merged**